src/gui/transferlistmodel.cpp

```diff
@@ -368,6 +578,13 @@ QColor TransferListModel::stateForeground(const BitTorrent::TorrentState state)
     return m_stateForegroundColors[state];
 }
 
+QString TransferListModel::statusString(const BitTorrent::TorrentState state, const QString &errorMessage) const
```

Comment:
Curious, but why didn't you make this a lambda too?

Reply:
leftover... changed.